### PR TITLE
Removing support for social surveys

### PR DESCRIPTION
**Pull request by warrenbailey**
# Motivation and Context

Removing social surveys from action, they are no longer run on RAS/RM and this will aid in our migration to the new system

# What has changed

# How to test?

# Links

### CODE REVIEW

**Review by jdfox — APPROVED** [review]

File: src/main/java/uk/gov/ons/ctp/response/action/service/decorator/CollectionExerciseAndSurvey.java [hunk 1]
```diff
@@ -23,16 +22,8 @@ public void decorateActionRequest(ActionRequest actionRequest, ActionRequestCont
...
-
-      if (context.getSampleUnitType() == SampleUnitDTO.SampleUnitType.H
-          && context.getAction().getActionType().getActionTypeNameEnum()
-              != uk.gov.ons.ctp.response.action.representation.ActionType.SOCIALICF) {
-        dateFormat = new SimpleDateFormat(ActionProcessingService.DATE_FORMAT_IN_SOCIAL_LETTER);
-      } else {
-        dateFormat = new SimpleDateFormat(ActionProcessingService.DATE_FORMAT_IN_REMINDER_EMAIL);
-      }
+      DateFormat dateFormat =
+          new SimpleDateFormat(ActionProcessingService.DATE_FORMAT_IN_REMINDER_EMAIL);
```
Comment (akssinhaamit) [inline src/main/java/uk/gov/ons/ctp/response/action/service/decorator/CollectionExerciseAndSurvey.java:26]: should it be reminder emails only?

**Review by jterryons — APPROVED** [review]

**Pull request merged**